backend/test/routes/Terraform.test.ts

```diff
@@ -50,11 +52,18 @@ describe('/terraform/resources endpoint', () => {
             .reply(200, readFileSync(resolve(__dirname, 'fixtures/terraform/template.zip'), null));
 
         const response = await request(app)
-            .post('/console/terraform/resources?zipUrl=http://test/test.zip')
+            .post(`${endpointUrl}?templateUrl=http://test/test.zip`)
```

Comment:
Optional:

Maybe a small improvement here 🙂 
```suggestion
            .post(endpointUrl)
            .query({ templateUrl: 'http://test/test.zip' })
```

Similar can be applied in L65.

Reply:
Sure, seems neat!
Thanks for pointing that out! 🚀